docs/_includes/footer.html

```diff
@@ -5,12 +5,23 @@
 
         {% if page.url != 'search/index' %}
         <div class="block block__sub">
-            <small>
-                <strong>Page last edited:</strong>
-                <time datetime="{{ page.last_modified_at }}">
-                    {{ page.last_modified_at | date: "%l:%M:%S %p %b %d, %Y" }}
-                </time>
-            </small>
+            <p>
+                <small>
+                    <strong>Help improve this page:</strong>
+                    <a href="https://github.com/cfpb/design-system/issues/new?assignees=&labels=content%2C+help+wanted&template=component-needs-content-edits.md&title=Suggestions+for%3A+%22{{ page.title }}%22">
```

Comment:
Keep it repo agnostic:

```suggestion
                    <a href="https://github.com/{{ site.repository }}/issues/new?assignees=&labels=content%2C+help+wanted&template=component-needs-content-edits.md&title=Suggestions+for%3A+%22{{ page.title }}%22">
```